modules/fw/manifests/networks.pp

```diff
@@ -36,18 +36,10 @@
     # AWS Networks; See https://github.com/mozilla-releng/build-cloud-tools/blob/master/configs/subnets.yml
 
     # USE1 Network CIDRs
```

Comment:
We may want to verify that these are buildbot-worker-only subnets... i.e., no taskcluster workers.

Reply:
If these subnets are verified not be in use anymore, we should make sure to delete them from the vpc

Reply:
aki, I've checked and they don't overlap. In use1, signing scriptworkers are all in 10.134.30.0/24, and non-signing scriptworkers in 10.134.48.0/24; in usw2, all scriptworkers are in 10.132.30.0/24.

dividehex, and update cloud-tools I guess. More than I was planning on right now, I'll see if I can get to it.